### PR TITLE
[level 2] Migrate section label to emotion

### PR DESCRIPTION
**Pull request by RichardPK**
psammead-section-label

Relates to https://github.com/bbc/psammead/issues/3763.

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-section-label/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 7.0.0 | [?](https://github.com/bbc/psammead/pull/3883) Emotion migration |
```
Comment (simonsinclair) [inline packages/components/psammead-section-label/CHANGELOG.md]:
```suggestion
| 7.0.0 | [PR#3886](https://github.com/bbc/psammead/pull/3883) Emotion migration |
```

File: packages/components/psammead-section-label/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 7.0.0 | [PR#3886](https://github.com/bbc/psammead/pull/3883) Emotion migration |
```
Comment (ryanmccombe) [inline packages/components/psammead-section-label/CHANGELOG.md]:
```suggestion
| 7.0.0 | [PR#3886](https://github.com/bbc/psammead/pull/3886) Emotion migration |
```

File: packages/components/psammead-section-label/src/index.jsx [hunk 3]
```diff
@@ -48,9 +48,9 @@ const SectionLabelWrapper = styled.div`
   }
 
   ${({ visuallyHidden }) =>
+    // Hide for all breakpoints
```
Comment (ryanmccombe) [inline packages/components/psammead-section-label/src/index.jsx]: Can we have comments in css / styled block?

**Review by ryanmccombe — APPROVED** [review]

File: packages/components/psammead-section-label/src/index.jsx (hunk 3 above)
Comment (RichardPK) [reply packages/components/psammead-section-label/src/index.jsx]: I'll bin

**Pull request merged**